jest.config.cjs

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const nextJest = require("next/jest");
+/**
+ * Setting up Jest
+ * https://nextjs.org/docs/testing#setting-up-jest-with-the-rust-compiler
+ *  */
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: "./",
+});
+
+// Add any custom config to be passed to Jest
+/** @type {import('jest').Config} */
+const customJestConfig = {
+  // Add more setup options before each test is run
+  // setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  transformIgnorePatterns: ["node_modules/(?!(react-markdown))"],
```

Comment:
saw this on the docs "Ignoring node_modules from test resolving and transforms"
does this mean it would get configured automatically under the hood ?
https://nextjs.org/docs/testing#setting-up-jest-with-the-rust-compiler

Reply:
or is this a specific requirement for react-markdown?

Reply:
There's an issue with ESM + Jest that React-markdown and Jest exhibit. You got great questions, see below :)